src/components/primitive/Icons/icon.component.props.ts

```diff
@@ -0,0 +1,11 @@
+import { IconType } from './icons';
+
+interface IconProps extends React.ImgHTMLAttributes<HTMLImageElement> {
+  icon: IconType;
+  size?: number;
+  color?: string;
+
+  default?: any;
```

Comment:
Don't use default as a prop name since it's a keyword.

Reply:
removed this, will not use it in the future.